### ISSUE TITLE
build_bins.sh  undefined method `options' for #<Cabin::Subscriber> (NoMethodError)

### ISSUE THREAD

**Issue by pu239ppy** [bloomberg/chef-bach#357]
There is a fix in the works
https://groups.google.com/forum/#!topic/fpm-users/KiaVXyqf6dQ

For manual workaround uninstall the cabin gem and install cabin v.0.7.2

**Comment by cbaenziger** [comment]
Until the community fixes this Lev pointed out we can:
````
vagrant@bcpc-bootstrap:~/chef-bcpc$ sudo gem uninstall cabin
vagrant@bcpc-bootstrap:~/chef-bcpc$ source proxy_setup.sh
vagrant@bcpc-bootstrap:~/chef-bcpc$ sudo -E gem install cabin -v0.7.2
````

**Comment by pu239ppy** [comment]
As far as we are concerned this has been fixed in #364
